### ISSUE TITLE
request() doesn't work

### ISSUE THREAD

**Issue by mc-y-saito** [mtownsend5512/request-xml#2]
Hi, I have an issue for using request-xml as I write code as
`$xml = request()->xml();` because `request()` doesn't have any values. It returns null.

Instead, `$request` is an object which contains request-header, body and so on. 
How can I utilize your good tool? Am I making wrong something...? 
I appreciate hearing from you.

**Comment by mtownsend5512** [comment]
Can you give an example of a request you are making to your Laravel app? This package detects if XML is sent, converts the XML into an array format and merges it into the Request object. You also need to have the Middleware active on the route that is receiving the XML. You can set it globally like so:

    protected $middleware = [
        \Mtownsend\RequestXml\Middleware\XmlRequest::class,
    ];

**Comment by mc-y-saito** [comment]
Thanks for your reply.
Honestly, I finally solved this by doing it my self.
I converted xml request by json_encode/decode and so on, so I don't have any issue for that.

But Yes, I have sent post request from Postman, its body is XML document like `<?xml version="1.0" encoding="UTF-8"?>
<DATA>
    <ID>foo</ID>
    <PASS>bar</PASS>
    <VALUE>foobar</VALUE>
</DATA>`.
I tried to get the request by `$xml = request()->xml();` but didn't work well for my environment. 
And of course I enabled the middleware setting.
